http_service/pkg/http_user_service/endpoint_test.go

```diff
@@ -0,0 +1,50 @@
+package httpuserservice
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/go-kit/log"
+	"github.com/jcromanu/final_project/http_service/pkg/entities"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func TestMakeCreateHTTPUserEndpooint(t *testing.T) {
+	serviceMock := new(HTTPServiceMock)
+	logger := log.NewLogfmtLogger(os.Stderr)
+	testCases := []struct {
+		testName       string
+		input          createUserRequest
+		expectedOutput int32
+		expectedError  error
+	}{
+		{
+			testName:       "endpoint valid user",
+			input:          createUserRequest{entities.User{Name: "Juan", Pwd_hash: "ooo", Age: 31, Additional_information: "no info", Parent: []string{}}},
+			expectedOutput: 1,
+			expectedError:  nil,
+		},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.testName, func(t *testing.T) {
+			ctx := context.Background()
+			assert := assert.New(t)
+			inputRequest := tc.input
+			serviceMock.On("CreateUser", mock.Anything, mock.Anything).Return(1, nil)
```

Comment:
This could be defined specifically for each test case. If you add a user not found test case this does not work.

Reply:
I´m rewriting all test cases to substitute the test suite for test tables